src/main/kotlin/interpreter/pyobject/types/PyTupleType.kt

```diff
@@ -27,10 +27,7 @@ import green.sailor.kython.interpreter.pyobject.PyType
  * Represents the type of a tuple (the `tuple` builtin).
  */
 object PyTupleType : PyType("tuple") {
-    override fun newInstance(args: Map<String, PyObject>): PyObject {
```

Comment:
For consistency newInstance should always be a full function, plus this will eventually be fleshed out into one with iterators.